scalingomock/api_mock.go

```diff
@@ -8,8 +8,8 @@ import (
 	http0 "net/http"
 	reflect "reflect"
 
-	go_scalingo "github.com/Scalingo/go-scalingo/v4"
-	http "github.com/Scalingo/go-scalingo/v4/http"
+	scalingo "github.com/Scalingo/go-scalingo"
```

Comment:
It requires the `v4`, gomock_generator is not yet clever enough, I've started the work on this, currently do it manually